data_fusion/plugins/modules/procflows/data_fusion.py

```diff
@@ -529,6 +529,11 @@ def call(fnames, command_line_args=None):
         command_line_args, {"METADATA": fuse_data["final"]["metadata_xobj"]}
     )
 
+    if command_line_args.get("image_compare_threshold") is not None:
+        image_compare_threshold = command_line_args["image_compare_threshold"]
+    else:
+        image_compare_threshold = None
```

Comment:
Replace with output_checker_kwargs.  Will be like  output_checker_kwargs["image"]["threshold"] = "medium"

Reply:
It looks to me like this has been fixed. Can one of you confirm @mindyls @evrose54 ?

Reply:
Is this tested/used at all @evrose54  ?  I can't remember the status

Reply:
It's been fixed. It's not used at all to my knowledge however.